Gopkg.toml

```diff
@@ -61,6 +65,10 @@
   name = "k8s.io/apiextensions-apiserver"
   version = "kubernetes-1.15.4"
 
+[[constraint]]
+  name = "sigs.k8s.io/controller-runtime"
+  version = "v0.3.0"
```

Comment:
Instead of using our own informer we are required to make use of the `controller-runtime` manager and controller now.

Reply:
E2E tests in this repo are now succeeding - do we know if there are any major changes / deprecations from our implementation?

Reply:
The e2e tests here were always green. I am not sure what you mean. Let's maybe hang and sort it out when this is a bigger/complicated topic.